app/controllers/samples_controller.rb

```diff
@@ -591,7 +591,7 @@ def bulk_upload_with_metadata
   def report_csv
     pipeline_run = select_pipeline_run(@sample, params[:pipeline_version])
     background_id = get_background_id(@sample, params[:background])
-    min_contig_reads = params[:min_contig_reads]
+    min_contig_reads = params[:min_contig_reads] || PipelineRun::MIN_CONTIG_READS
```

Comment:
Hey Greg, I notice that the `PipelineRunService.call` in `models/bulk_downloads.rb` to generate the Sample Taxon Report doesn't pass in anything for min_contig_reads. So that call would use a value of 0 while this call would use a value of 4.

Since we only load contigs into our database that have at least four contig reads (see `db_load_contigs` which uses `PipelineRun::MIN_CONTIG_READS`) any value from 0 to 4 would produce the same result. However, it still seems good to keep it consistent.

Perhaps we can make the default in PipelineReportService 4 instead of 0? And add a comment explaining that contigs less than 4 are filtered out anyways so people don't get confused. Curious what @tfrcarvalho thinks about changing the default.

Reply:
Thanks for pointing that out Mark. We definitely don't want bulk downloads to return different results than single reports.

I propose going further and eliminating the constant from pipeline report service. It does not appear to add any value. 

> Since we only load contigs into our database that have at least four contig reads

As a matter of fact, we filter out contigs less than 4 in the pipeline also. See

```
                    if unique_count < MIN_CONTIG_SIZE:
                        del contig_counts[contig]
```
And in another place in the pipeline, the value appears to come from idseq-web. 
```
        min_contig_size = self.additional_attributes.get("min_contig_size", MIN_CONTIG_SIZE)
```

As a side note, when we add constants that should be in sync across codebases, it is good to add a comment to that effect in the code.

Reply:
* I don't think web needs to be in sync with pipeline, in this case. More and more we should detach web from pipeline. If the pipeline makes a decision of not publishing contigs smaller than 4, so be it. Exceptions *could* be any parameter that is configurable through the web -> pipeline interface.

* The default should however be in sync throughout the web. In that case, i would prefer `0` which would publish any values that the pipeline decides 
publish (even if the value changes later).

Reply:
I agree, I don't think these constants should be in sync. I should have written "constants that should be **informed by** constants in other codebases".

So how we the values in idseq-web? PipelineReportService:: DEFAULT_MIN_CONTIG_READS and PipelineRun::MIN_CONTIG_READS?

Note that the method endpoint `summary_contig_counts` already has the expression `min_contig_reads = params[:min_contig_reads] || PipelineRun::MIN_CONTIG_READS` which is why I added it to `report_csv`.

Reply:
I replaced `DEFAULT_MIN_CONTIG_READS` with `PipelineRun::MIN_CONTIG_READS`.

Reply:
`PipelineRun::DEFAULT_MIN_CONTIG_READS` doesn't make sense because that constant is also used in `PipelineRun` in loading the contigs. While it may be good to remove that, that would be another PR.